interfaces/ip_router/v0/README.md

```diff
@@ -37,24 +37,25 @@ Both the Requirer and the Provider need to adhere to criteria to be considered c
 
 ```yaml
 requirer:
-  app: {
-    "networks": [
-      {
-        "network": "192.168.250.0/24",
-        "gateway": "192.168.250.1",
-        "routes": [
-          {
-            "destination": "172.250.0.0/16",
-            "gateway": "192.168.250.3"
-          }
-        ]
+  app:
+    "app-network": {
+      "networks":
+        {
+          "network": "192.168.250.0/24",
+          "gateway": "192.168.250.1",
+          "routes": [
+            {
+              "destination": "172.250.0.0/16",
+              "gateway": "192.168.250.3"
+            }
+          ]
+        }
```

Comment:
I think we can probably get rid of the `networks` key layer, as the network we are requesting is already completely defined by `app-network`. If we need more than one network, we'll have two keys like `app-network` and `other-network`.

Reply:
This is because it's easier to json dump and load the whole payload instead of having to loop over the keys that exist in the databag each time.